mobile/pages/Map.tsx

```diff
@@ -0,0 +1,105 @@
+import React, {useState, useEffect} from 'react';
+import { useFonts, Poppins_500Medium } from '@expo-google-fonts/poppins';
+import { vw, vh } from 'react-native-expo-viewport-units';
+import { StyleSheet, Text, TouchableOpacity, View } from 'react-native';
+import MapView, { Marker, Callout } from 'react-native-maps';
+
+
+
+
+export default function Map(props: any ){
+    useFonts({
+        Poppins_500Medium,
+    });
+
+
+const [counterMarker, setCounterMarker] = useState(0);
+const [markers = [], setMarkers] = useState();
+
+
+useEffect(()=> {
+    async function resMarkers(){
+        let markersTemp =  await props.route.params.body;
+        
+         setMarkers(await markersTemp)
+        
+    }
+
+    resMarkers();
+}, [])
+
+console.log("MAP:" + markers)
+
+
+
+    return(
+        <View style={css.container}>
+            <View style={css.header}>
+                <Text style={css.titlePage}>Bem-vindo</Text>
+                <Text style={css.description}>Encontre no mapa um ponto de coleta.</Text>
+            </View>
+
+            <MapView 
+            style={css.mapStyle}
+            initialRegion={{
+                latitude: -28.495696340879743,
+                longitude: -49.01386381255002,
+                latitudeDelta: 0.0922,
+                longitudeDelta: 0.0421,
```

Comment:
What do you think of extracting a global file variable?
global files variables follow this pattern 
```ts
const initialLatitude = -28.495696340879743;
```